mmv1/products/dataplex/Datascan.yaml

```diff
@@ -0,0 +1,640 @@
+                  default_value: 0
+                - !ruby/object:Api::Type::NestedObject
+                  name: 'rangeExpectation'
+                  description: |
+                    ColumnMap rule which evaluates whether each column value lies between a specified range.
+                  properties:
+                    - !ruby/object:Api::Type::String
+                      name: 'minValue'
+                      description: |
+                        The minimum column value allowed for a row to pass this validation. At least one of minValue and maxValue need to be provided.
+                    - !ruby/object:Api::Type::String
+                      name: maxValue
+                      description: |
+                        The maximum column value allowed for a row to pass this validation. At least one of minValue and maxValue need to be provided.
+                    - !ruby/object:Api::Type::Boolean
+                      name: 'strictMinEnabled'
+                      default_value: false
+                      description: |
+                        Whether each value needs to be strictly greater than ('>') the minimum, or if equality is allowed. 
+                        Only relevant if a minValue has been defined. Default = false.
+                    - !ruby/object:Api::Type::Boolean
+                      name: 'strictMaxEnabled'
+                      default_value: false
+                      description: |
+                        Whether each value needs to be strictly lesser than ('<') the maximum, or if equality is allowed.
+                        Only relevant if a maxValue has been defined. Default = false.
+                - !ruby/object:Api::Type::NestedObject
+                  name: 'nonNullExpectation'
+                  description: |
+                    ColumnMap rule which evaluates whether each column value is null.
+                - !ruby/object:Api::Type::NestedObject
+                  name: 'setExpectation'
+                  description: |
+                    ColumnMap rule which evaluates whether each column value is contained by a specified set.
+                  properties:
+                    - !ruby/object:Api::Type::Array
+                      name: 'values'
+                      description: |
+                        Expected values for the column value.
+                - !ruby/object:Api::Type::NestedObject
+                  name: 'regexExpectation'
+                  description: |
+                    ColumnMap rule which evaluates whether each column value matches a specified regex.
+                  properties:
+                    - !ruby/object:Api::Type::String
+                      name: 'regex'
+                      description: |
+                        A regular expression the column value is expected to match.
+                - !ruby/object:Api::Type::NestedObject
+                  name: 'uniquenessExpectation'
```

Comment:
we'll want the nested object defined here as well

Reply:
i believe this is an empty message, so it should be fine?